ruby_connectors/spec/salesforce_connector/client_spec.rb

```
@@ -0,0 +1,37 @@
+
+RSpec.describe RubyConnectors::SalesforceConnector::Client do
+  let(:valid_config) do
+    {
+      client_id: ENV['SALESFORCE_CLIENT_ID'],
+      client_secret: ENV['SALESFORCE_CLIENT_SECRET'],
+      refresh_token: ENV['SALESFORCE_REFRESH_TOKEN']
+    }
+  end
+
+  describe '#catalog' do
+    it 'returns schema information', vcr: { record: :once } do
```

RuboCop (1.73)

[convention] Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

(Style/StringLiterals)